tests/integration/targets/filter_json_patch/tasks/main.yml

```
@@ -0,0 +1,139 @@
+---
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: test json_patch
```

Task names should start with an upper-case letter:
```suggestion
- name: Test json_patch
```